tests/unit/test_task_utils.py

```diff
@@ -170,3 +171,17 @@ def test_write_to_json():
     write_to_json.run(dict, "dict.json")
     assert os.path.exists("dict.json")
     os.remove("dict.json")
+
+
+def test_df_clean_column():
```

Comment:
Add a new test to the second option - cleaning one/two columns not all column in DF